apis/r/configure

```diff
@@ -39,7 +39,7 @@ else
         cmake -B build -S libtiledbsoma -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/local -DOVERRIDE_INSTALL_PREFIX=OFF -DTILEDBSOMA_BUILD_R=ON > cmake_log.txt
         cmake --build build -j 2 >> cmake_log.txt
         sudo cmake --build build --target install-libtiledbsoma >> cmake_log.txt
-        rm -rf build
+        #rm -rf build
     	export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
```

Comment:
BTW that should not be needed (but may not do harm).  On Debian/Ubuntu and alike:

```sh
$ cat /etc/ld.so.conf.d/* | grep /usr/local/lib
/usr/local/lib/i386-linux-gnu
/usr/local/lib/i686-linux-gnu
/usr/local/lib
/usr/local/lib/x86_64-linux-gnu
$ 
```